escrow/README.md

```diff
@@ -7,11 +7,20 @@
 
 ## Overview
 
-TODO: need UI for this to be relevant
+An escrow is a neutral third party that holds an asset on behald of two parties until a transaction has occured. Once the transaction has taken place the escrow can be resolved and the assets will be transferred as dictated by the contract. 
+
+Outside of the blockchain the asset can be pretty much anything such as currency which will be used as a payment for a product or perhaps a deed to an estate. In the blockchain world it all comes down to 1s and 0s however those assets can be tokenized and logic can be coded into a contract to handle those cases but it still comes down to good faith outside of the blockchain to carry out the obligation.
+
+In this particular case the asset can be any native asset on the Fuel network and the transaction is considered to be between a buyer and a seller. At the moment, and for the foreseeable future, there is another party (the arbiter) in case there is a dispute that the buyer and seller cannot resolve amongst themselves. More information can be found in the [specification](./SPECIFICATION.md).
```

Comment:
```suggestion
For this application, the on-chain asset can be any native asset on the Fuel Network and the transaction is considered to be between a buyer and a seller. At the moment, and for the foreseeable future, there is another party (the arbiter) in case there is a dispute that the buyer and seller cannot resolve amongst themselves. More information can be found in the [specification](./SPECIFICATION.md).
```